### ISSUE TITLE
Quest Bug:  Shutting Down Manaforge B'naar

### ISSUE THREAD

**Issue by rykon1459918** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3809]
(Please note I did try to already log this but I may of screwed up apology if this is a duplicate but I could not find my original at all)
Report ID:

Reporter Name: 
Tureon / Zueron (In game names)

Date/Time: 
06-June-2016 Multiple times.

Error Severity: 
Minor / Medium

Zone/Instance:
NetherStorm

What type of error occurred?:
Unable to complete the quest Shutting Down Manaforge B'naar BUT only if the quest has already been completed or attempted already on this server restart.
The first time you attempt to finish this quest each server restart it works fine, I was able to get full credit on my main after multiple attempts, however this was only after camping the area after the last server restart, When the quest has already been attempted (either attempted and failed or completed) the console you need to access (labeled on live as B'naar Control Console) once you get the key off the forgemaster does not seem to respawn at all.
I attempted this multiple times on multiple characters, including on purposely running in and failing the quest (not defending it for 2 minutes) and the forge did not respawn I checked every few hours and even over multiple days.

What was the actual result vs. the expected result?: If you are not the first person to attempt the quest on server restart there is no actual clickable interface for the B'naar Control Console. VS on live I went and tested this the respawn of the click-able section of the manaforge (B'naar Control Console) was about 2-3 minutes.

Can the error be repeated? If so, what are the steps?: Simply have a user complete the quest the first time around then have a second user with the quest try, it will fail as the clickable part of the B'naar Control Console doesn't seem to respawn.

Link to Video Evidence: N/A don't have the ability to record

External/Related Links: 
http://www.wowhead.com/quest=10329/shutting-down-manaforge-bnaar#comments

ANOTHER NOTE: Had a friend also try the Aldor version of this chain (as I went with scyer) he had the exact same results.

**Comment by devidao** [comment]
it was fixed on another issue, but looks like still not applied on PTR.

**Comment by rykon1459918** [comment]
Ok thanks for the update, sorry  I did try to search for threads first before posting.
